test/assets/messages/AcceptanceDateMessages.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package messages
+
+object AcceptanceDateMessages extends BaseMessages {
+
+  val title: String = "Was the entry acceptance date before 1 January 2021?"
+  val h1: String = "Was the entry acceptance date before 1 January 2021?"
+  val radioButtonOne: String = "Yes"
```

Comment:
These will become site.yes and site.no so might be worth moving to the Base Messages file.

Reply:
removed acceptanceDate.yes and acceptanceDate.no from Base Messages file and added site.yes and site.no